src/components/sidebar.js

```diff
@@ -120,6 +121,8 @@ const Sidebar = () => (
         <NavLink to="/sdk/store/">Store Endpoint</NavLink>
         <NavLink to="/sdk/envelopes/">Envelopes</NavLink>
         <NavLink to="/sdk/event-payloads/" title="Event Payloads">
+          <NavLink to="/sdk/event-payloads/transaction/">Transaction Type</NavLink>
+          <NavLink to="/sdk/event-payloads/span/">Span Interface</NavLink>
```

Comment:
Looking at how this will be rendered, it is strange to have Transaction and Span above all other Interfaces that are part of regular (error) events.

_Since this is related to the bigger difficulty in explaining Events x Transactions and what are valid attributes in each, feel free to ignore me..._